apiHelpers/atelier.js

```diff
@@ -1 +1,61 @@
-const endpoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/:hr-la/'
\ No newline at end of file
+const endpoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-lax'; //note that all endpoints must contain a '/' after 'hr-lax'
+const credentials = require('../config.js')
+// import credentials from '../config.js'
+const axios = require ('axios');
+
+module.exports = {
+  getProducts: function(callback) {
+    axios({
+      method: 'GET',
+      url: `${endpoint}/products`,
+      headers: { 'Authorization': credentials.TOKEN },
+    })
+    .then(res => {
+      console.log(res.data)
+      callback(null, res.data)
```

Comment:
continuing comment above - the callback is unnecessary to pass information. by simply doing a 'return res', we can avoid using any sort of callback here. If there is an error, the .catch block will catch it.